File: ASP.NET MVC Editores Date y DateTime con helper para gestión de scripts/[C#]-ASP.NET MVC Editores Date y DateTime con helper para gestión de scripts/C#/MVCDateTimePicker/Models/Persona.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace MVCDateTimePicker.Models
{
    public class Persona
    {
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public string Ciudad { get; set; }
        [Display(Name="Fecha de Nacimiento")]
        [DataType(DataType.Date)]
        public DateTime FechaNacimiento { get; set; }
        [Display(Name="Último Acceso")]
        public DateTime UltimoAcceso { get; set; }
        [Display(Name="Fecha de Registro")]
        [DataType(DataType.Date)]
        public DateTime FechaRegistro { get; set; }
        [Display(Name = "Última Operación")]
        public DateTime UltimaOperacion { get; set; }
    }
}

File: ASP.NET MVC Editores Date y DateTime con helper para gestión de scripts/[VB]-ASP.NET MVC Editores Date y DateTime con helper para gestión de scripts/VB/MVCDateTimePicker/Models/Persona.vb
Imports System.ComponentModel.DataAnnotations

Public Class Persona
    Property Nombre As String
    Property Apellido As String
    Property Ciudad As String
    <Display(Name:="Fecha de Nacimiento")> _
    <DataType(DataType.Date)> _
    Property FechaNacimiento As DateTime
    <Display(Name:="Último Acceso")> _
    Property UltimoAcceso As DateTime
    <Display(Name:="Fecha de Registro")> _
    <DataType(DataType.Date)> _
    Property FechaRegistro As DateTime
    <Display(Name:="Última Operación")> _
    Property UltimaOperacion As DateTime
End Class
